src/components/Broadcast/Broadcast.style.ts

```diff
@@ -0,0 +1,137 @@
+import { css } from '@emotion/react';
+import { rem } from 'polished';
+import type { Theme } from 'theme';
+
+import type { BroadcastProps } from './Broadcast.types';
+import { generateStylesFromTokens } from '../Typography/utils';
+
+const broadcastStyles = (props: BroadcastProps) => (theme: Theme) =>
+  css`
+    --_border-width: ${theme.dimension.borderWidth.get('default')};
+    --_outline-width: ${theme.dimension.borderWidth.get('focused')};
+    --_min-height: ${rem(44)};
+    --_min-content-width: ${rem(100)};
+
+    box-sizing: border-box;
+    position: relative;
+    isolation: isolate;
+    padding: calc(${theme.dimension.spacing.get('md')} - var(--_border-width))
+      calc(${theme.dimension.spacing.get('md')} - var(--_border-width));
+    display: inline-grid;
+    grid-template-columns: auto minmax(var(--_min-content-width), 1fr) auto;
+    grid-template-areas: 'icon content actions close';
+    align-items: center;
+    background: ${theme.tokens.colors.get('palette.secondary.muted')};
+    border-radius: ${theme.dimension.borderRadius.get('md')};
+    border: 1px solid ${getBorderColor(props.status, theme)};
```

Comment:
why `1px` and not `var(--_border-width)` ?

Reply:
Good catch we missed this in `InlineAlert` as well. I will update it in both places.